File: gems/decomplex/examples/java/function-lcom.java
class Example { static Result mixed(int price, int tax, Logger logger) {
  var subtotal = price + tax;
  var total = subtotal * 2;
  var rounded = total.round();

  var timestamp = now();
  var buffer = Buffer.init();
  var stamp = timestamp;
  buffer.push(stamp);
  logger.info(buffer);

  return Result.init(rounded, buffer);
} }

File: gems/decomplex/examples/cpp/function-lcom.cpp
Result mixed(int price, int tax, Logger logger) {
  auto subtotal = price + tax;
  auto total = subtotal * 2;
  auto rounded = total.round();

  auto timestamp = now();
  auto buffer = Buffer.init();
  buffer.push(timestamp);
  logger.info(buffer);

  return Result.init(rounded, buffer);
}
